static/app/utils/replays/replayReader.tsx

```diff
@@ -419,6 +420,18 @@ export default class ReplayReader {
 
   getLPCFrames = memoize(() => this._sortedSpanFrames.filter(isLCPFrame));
 
+  getVideoAttachments = memoize(() =>
+    (
+      this._sortedRRWebEvents.filter(
+        event => event.type === EventType.Custom && event.data.tag === 'video'
+      ) as VideoFrameEvent[]
+    ).map(event => ({
+      duration: event.data.payload.duration,
+      id: event.data.payload.segmentId,
+      timestamp: event.timestamp,
+    }))
```

Comment:
using `event is VideoFrameEvent` instead of casting
```suggestion
    this._sortedRRWebEvents
      .filter(
        (event): event is VideoFrameEvent =>
          event.type === EventType.Custom && event.data.tag === 'video'
      )
      .map(event => ({
        duration: event.data.payload.duration,
        id: event.data.payload.segmentId,
        timestamp: event.timestamp,
      }))
```